helios/pipeViewer/pipe_view/core/src/core.pyx

```diff
@@ -406,7 +406,8 @@ cdef class Renderer:
         return string_to_display, wx.TheBrushList.FindOrCreateBrush(wx.WHITE, wx.SOLID)
 
     def setFontFromDC(self, dc):
-        self.c_font = getFont(dc.GetFont())[0]
+        py_font = dc.GetFont()
+        self.c_font = getFont(py_font)[0]
```

Comment:
Quick explainer of what's happening here:
* `dc.GetFont()` returns a Python `wx.Font` object
* We call `getFont` to unwrap it and grab a pointer to the C++ `wxFont` object underneath it
* We dereference that pointer and store a copy of the object in this class

The latest version of Cython decided it was a good idea to garbage collect the Python object returned from `dc.GetFont()` right before dereferencing the C++ pointer and try to copy the C++ object, which means we end up trying to make a copy of a freed object and we get a segfault.

By storing the result of `dc.GetFont()` in a temporary variable, we can ensure it lives long enough for the copy to succeed.